app_dart/lib/src/service/luci.dart

```diff
@@ -229,6 +229,7 @@ class LuciBuilder {
     this.enabled,
     this.runIf,
     this.taskName,
+    this.benchmark,
```

Comment:
Changes to this file should not be necessary at this time. Generally, only commit changes that are going to be used in this PR